frontend/pulse3d/src/components/layouts/ControlPanel.js

```diff
@@ -190,6 +191,11 @@ export default function ControlPanel() {
 
           const handleSelected = (e) => {
             e.preventDefault();
+            //if metric definitions button then open window with metric definitions
+            if (label === userButtons[3].label) {
+              window.open("https://pulse3d.readthedocs.io/en/latest/gettingstarted.html#jupyter-notebooks");
```

Comment:
@jperson @luciipak is there anything in the CSP that would prevent this from opening? I'm assuming the answer is no right now since Nikita posted a video of it working, but just wondering if we'll need to add this to the CSP in the future

Reply:
This is also on the test cluster